packages/contracts-bedrock/contracts/L1/OptimismPortal.sol

```diff
@@ -361,7 +425,7 @@ contract OptimismPortal is Initializable, ResourceMetering, Semver {
         uint64 _gasLimit,
         bool _isCreation,
         bytes memory _data
-    ) public payable metered(_gasLimit) {
+    ) public payable whenNotPaused metered(_gasLimit) {
```

Comment:
I have a similar (but stronger) reaction to the comment on `proveWithdrawalTransaction`, ie. I can easily imagine wanting to pause withdrawals but not deposits. 

Not that hard to implement if we can agree on the value of doing so.

Reply:
Going to remove this

Reply:
Pardon me for overriding, but just to be safe I'd prefer to resolve once it's done :)